src/components/image/image.tsx

```diff
@@ -111,8 +107,17 @@ export class Image
     ];
     return body;
   }
+
+  private shouldLazyLoad(): boolean {
+    const img: HTMLImageElement = this.element.querySelector("img");
+    return this.lazyLoad && (!img || img.classList.contains(LAZY_LOAD_CLASS));
```

Comment:
Minor suggestion: We could return immediately if lazyLoad is false. So we do not querySelector unnecessary.

Reply:
Great catch! Thanks!